config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -0,0 +1,135 @@
+periodics:
+- name: ci-npd-build
+  interval: 2h
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: node-problem-detector
+    base_ref: master
+    path_alias: k8s.io/node-problem-detector
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-master
+      command:
+      - runner.sh
+      args:
+      - make
+      - build
+
+- name: ci-npd-test
+  interval: 2h
+  decorate: true
+  labels:
+    preset-service-account: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: node-problem-detector
+    base_ref: master
+    path_alias: k8s.io/node-problem-detector
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-master
+      command:
+      - runner.sh
+      args:
+      - make
+      - test
+
+- name: ci-npd-e2e-node
+  interval: 2h
+  decorate: true
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: node-problem-detector
+    base_ref: master
+    path_alias: k8s.io/node-problem-detector
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190220-615bdbe2e-master
+      command:
+      - runner.sh
+      args:
+      - bash
+      - -c
+      - |
+        make push-ci &&
```

Comment:
I think in CI it's a waste of resource? You can have a periodic build job and let all jobs just use the latest npd?

Reply:
Updated